docs/schema-loader-import.md

```diff
@@ -0,0 +1,271 @@
+# Importing existing tables to ScalarDB using ScalarDB Schema Loader
+
+You might want to use ScalarDB (e.g., for database-spanning transactions) with your existing databases. In that case, you can import those databases under the ScalarDB control using ScalarDB Schema Loader. ScalarDB Schema Loader automatically adds ScalarDB-internal metadata columns in each existing table and metadata tables to enable various ScalarDB functionalities including transaction management across multiple databases.
+
+## Before you begin
+
+{% capture notice--warning %}
+**Attention**
+
+You should carefully plan to import a table to ScalarDB in production because it will add transaction metadata columns to your database tables and the ScalarDB metadata tables. There would also be several differences between your database and ScalarDB and limitations.
+{% endcapture %}
+
+<div class="notice--warning">{{ notice--warning | markdownify }}</div>
+
+### What will be added to your databases
```

Comment:
I think we should talk about how much importing is lightweight, like, the time for importing doesn't take proportionally to the database size and usually takes several seconds.

Reply:
Fixed in e3dd00d.